src/webui/webapplication.h

```diff
@@ -26,95 +26,167 @@
  * exception statement from your version.
  */
 
-#ifndef WEBAPPLICATION_H
-#define WEBAPPLICATION_H
+#pragma once
```

Comment:
I'm not against it, but I think you should add a rule in our coding guidelines about it.

Reply:
The current situation is no one is against that, but most developers (except me, generally) use include guards (maybe just habit?). Should we require "pragma once" by creating the appropriate rules? Then the dispute between supporters and opponents may inevitably arise. I'm afraid if I try to insist on this, it will generally be prohibited in favor of the include guard.

Reply:
>I'm afraid if I try to insist on this, it will generally be prohibited in favor of the include guard.

Okay, just to be clear, my viewpoint is, having different include guards in the same project is distributing, having a rule for it will clear the confusion. 

>Then the dispute between supporters and opponents may inevitably arise.

I think the worst scenario is just we make a vote/decision and we move on, me at least won't bother fight hard for it.

Reply:
Ok, just removed it here for now to not slow this PR. Let it be another issue.